frontend/src/modules/history/actions.js

```diff
@@ -117,10 +117,24 @@ export function updateStatus(
 }
 
 
+export function deleteTranslation(
+    entity: number,
+    locale: string,
+    pluralForm: number,
+    translation: number,
+): Function {
+    return async dispatch => {
+        await api.translation.delete(translation);
```

Comment:
Cool thing about reviewing this code: it makes me realize we don't have any sort of tolerance to errors, and we should. For example, here if the server returns any kind of error, the client will throw an error as well, and that error never being caught will propagate and make the client stop running. It's completely out of scope here, but in the future we should add some try / catch blocks around all our API calls somehow, and show some errors to the user. I might do that as part of [bug 1500871](https://bugzilla.mozilla.org/show_bug.cgi?id=1500871).

Reply:
Agreed, code-wise this is very much related to that bug. Adding a reminder comment there.